feeds_jsonpath_parser.install

```diff
@@ -37,12 +37,12 @@ function feeds_jsonpath_parser_requirements($phase) {
     if (module_exists('libraries') && file_exists(libraries_get_path('jsonpath') . '/jsonpath.php')) {
       $message = $t('To remain secure because of issues identified in <a href="https://www.drupal.org/sa-contrib-2019-083">SA-CONTRIB-2019-083</a>, you must remove goessner/jsonpath and replace it with flow/jsonpath.');
     }
-    if ($phase === 'runtime' || (module_exists('libraries') && !file_exists(libraries_get_path('jsonpath') . '/src/Flow/JSONPath/JSONPath.php'))) {
+    if ($phase === 'runtime' || (module_exists('libraries') && !file_exists(libraries_get_path('jsonpath') . '/src/JSONPath.php'))) {
```

Comment:
this is good (for this PR) but reminds me that this whole section needs a lot of work...